en/docs/samples/risk-based-adaptive-auth.md

```diff
@@ -0,0 +1,169 @@
+
+1.  Click **Service Providers > List**.
+
+2.  Click **Edit** on the `saml2-web-app-pickup-dispatch.com` service provider.
+
+3.  Expand the **Local and Outbound Configuration** section.
+
+4.  Click **Advanced Authentication**.
+
+5.  Expand **Script Based Conditional Authentication**.
+
+6.  Click **Templates** on the right side of the **Script Based Conditional Authentication** field and then click **Risk-Based**. 
+
+    ![tenant based template](../assets/img/samples/login-attempts-based-template.png)
+
+7.  Click **Ok**. The authentication script and authentication steps
+    are configured. 
+    
+    The authentication script defines a conditional step that executes the second step of authentication (the hardware key authenticator) if the riskScore is greater than 0.
+
+8.  The authentication steps added are `totp` and `fido`. However, these are authentication steps that you would normally use in production. 
+
+    To try out sample authenticators with the sample application, delete the two
+    authenticators and add the following sample authenticators instead.
+
+    1.  Click **Delete** to remove the `totp` authenticator from Step 2 (the
+        second authentication step).
+        
+        ![delete authenticator](../assets/img/samples/delete-authenticator-1.png)
+        
+    2.  Select **Demo Hardware Key Authenticator** and click **Add**.  
+        ![add new authenticator](../assets/img/samples/add-new-authenticator.png)
+
+9. Click **Update**.
+
+----
+
+## Try it out
+
+1.  Start the Tomcat server and access the following sample PickUp
+    application URL:
+
+    <http://localhost.com:8080/saml2-web-app-pickup-dispatch.com> 
+
+2.  Port offset the 9443 port in WSO2 Stream Processor. This is required because WSO2 IS also runs on the 9443 port.
+
+    1.  Open the `deployment.yaml` file found in the `<SP_HOME>/conf/worker/` folder.
+
+    2.  Change the `Port: 9443` to **`9444`** under `listenerConfigurations` .  
+
+        ``` java
+        listenerConfigurations:
+          -
+		   id: "default"
+		   host: "0.0.0.0"
+		   port: 9090
+          -
+           id: "msf4j-https"
+           host: "0.0.0.0"
+           port: 9444
+		   scheme: https
+	       keyStoreFile: "${carbon.home}/resources/security/wso2carbon.jks"
+		   keyStorePassword: wso2carbon
+		   certPass: wso2carbon
+        ```
+
+3.  Start the WSO2 SP server in a **Worker** profile.  
+    -   For Windows: `worker.bat`
+    -   For Linux: ./ `worker.sh`
+
+4.  Log in by giving username and password credentials (e.g., admin/admin). You are logged in to the application.
+
+    Note that the user is authenticated with basic authentication only.
+
+5.  Log out of the application.
+
+6.  Execute the following cURL command. This command publishes an event
+    about a user bank transaction exceeding $10,000.
+
+    !!! tip
+    
+        Replace the `<username>` tag in the cURL command given below with a valid username and ensure that the WSO2 SP Worker
```

Comment:
Make sure the following phrase is aligned (in review mode, it isn't)

Worker profile is running.